packages/table/src/utils/tableHelpers.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { PropertyFilterProps, TableProps as AWSUITableProps } from '@awsui/components-react';
+import { StatusIcon } from '@synchro-charts/core';
+import { round } from '@iot-app-kit/core';
+import { ColumnDefinition, TableItem } from './types';
+import { getIcons } from './iconUtils';
+import { LoadingSpinner } from './spinner';
+import './style.css';
+
+export const getDefaultColumnDefinitions: (
+  columnDefinitions: ColumnDefinition[]
+) => (AWSUITableProps.ColumnDefinition<TableItem> & ColumnDefinition)[] = (columnDefinitions) => {
+  return columnDefinitions.map((colDef) => ({
+    cell: (item: TableItem) => {
+      if (!(colDef.key in item)) {
+        return <div>-</div>;
```

Comment:
does this need to return a div? Could this just return a fragment `<>-</>` instead?

Reply:
Removed.